src/Carting.Api/Mappers/V2/Mapper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Carting.Api.Requests.V2;
+using Carting.Api.Responses.V2;
+
+namespace Carting.Api.Mappers.V2
+{
+    public static class Mapper
```

Comment:
There are two classes of mapper, on is for requests/responses and the other is for database access, consider naming a bit more explicit if it's static class and not rely on namespaces only, or just use already existing mapper libs that can do this for you